[CTOR-1031] Refresh network-nortel-standard-snmp doc page (#3763)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-nortel-standard-snmp.md

id: network-nortel-standard-snmp
title: Nortel Standard
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Pack assets

### Templates

The Monitoring Connector **Nortel Standard** brings a host template:

* **Net-Nortel-Standard-SNMP-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="Net-Nortel-Standard-SNMP-custom" label="Net-Nortel-Standard-SNMP-custom">

| Service Alias   | Service Template                                | Service Description                                                                                           |
|:----------------|:------------------------------------------------|:--------------------------------------------------------------------------------------------------------------|

> | Cpu             | Net-Nortel-Standard-Cpu-SNMP-custom             | Check the rate of utilization of CPU for the machine. This check can give the average utilization rate of CPU |

Reviewer: ```suggestion
| Cpu             | Net-Nortel-Standard-Cpu-SNMP-custom             | Check the rate of CPU utilization for the machine. This check can give the average CPU  utilization rate |
```

| Hardware-Global | Net-Nortel-Standard-Hardware-Global-SNMP-custom | Check hardware status                                                                                         |
| Memory          | Net-Nortel-Standard-Memory-SNMP-custom          | Check memory usage                                                                                            |
| Stack           | Net-Nortel-Standard-Stack-SNMP-custom           | Check stack units                                                                                             |

> The services listed above are created automatically when the **Net-Nortel-Standard-SNMP-custom** host template is used.

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias              | Service Template                                           | Service Description                                             | Discovery  |
|:---------------------------|:-----------------------------------------------------------|:----------------------------------------------------------------|:----------:|

> | Packet-Errors-Generic-Id   | Net-Nortel-Standard-Packet-Errors-Generic-Id-SNMP-custom   | Check packets on errors/discards of a network interface         |            |

Reviewer: ```suggestion
| Packet-Errors-Generic-Id   | Net-Nortel-Standard-Packet-Errors-Generic-Id-SNMP-custom   | Check packets in error/discarded from a network interface         |            |
```

> | Packet-Errors-Generic-Name | Net-Nortel-Standard-Packet-Errors-Generic-Name-SNMP-custom | Check packets on errors/discards of a network interface         |            |

Reviewer: ```suggestion
| Packet-Errors-Generic-Name | Net-Nortel-Standard-Packet-Errors-Generic-Name-SNMP-custom | Check packets in error/discarded from a network interface          |            |
```

> | Packet-Errors-Global       | Net-Nortel-Standard-Packet-Errors-Global-SNMP-custom       | Check packets on errors/discards of multiple network interfaces | X          |

Reviewer: ```suggestion
| Packet-Errors-Global       | Net-Nortel-Standard-Packet-Errors-Global-SNMP-custom       | Check packets in error/discarded from multiple network interfaces | X          |
```

> | Traffic-Generic-Id         | Net-Nortel-Standard-Traffic-Generic-Id-SNMP-custom         | Check traffic of an network interface                           |            |

Reviewer: ```suggestion
| Traffic-Generic-Id         | Net-Nortel-Standard-Traffic-Generic-Id-SNMP-custom         | Check traffic of a network interface                           |            |
```

> | Traffic-Generic-Name       | Net-Nortel-Standard-Traffic-Generic-Name-SNMP-custom       | Check traffic of an network interface                           |            |

Reviewer: ```suggestion
| Traffic-Generic-Name       | Net-Nortel-Standard-Traffic-Generic-Name-SNMP-custom       | Check traffic of a network interface                           |            |
```
